<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="129088" started-at="2020-10-10T18:15:59 EET" finished-at="2020-10-10T18:18:08 EET">
    <groups>
    </groups>
    <test name="Default test" duration-ms="129088" started-at="2020-10-10T18:15:59 EET" finished-at="2020-10-10T18:18:08 EET">
      <class name="testlogic.SignUpTest">
        <test-method status="PASS" signature="OpenDriver()[pri:0, instance:testlogic.SignUpTest@5f282abb]" name="OpenDriver" is-config="true" duration-ms="13623" started-at="2020-10-10T18:15:46 EET" finished-at="2020-10-10T18:15:59 EET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OpenDriver -->
        <test-method status="PASS" signature="UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb]" name="UserSignUpWithValidData" duration-ms="38119" started-at="2020-10-10T18:16:01 EET" data-provider="ExcelData" finished-at="2020-10-10T18:16:39 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[Eman]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Diab]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456.0]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[eman671@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[pass5678]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UserSignUpWithValidData -->
        <test-method status="PASS" signature="screenshotOnFailure(org.testng.ITestResult)[pri:0, instance:testlogic.SignUpTest@5f282abb]" name="screenshotOnFailure" is-config="true" duration-ms="12" started-at="2020-10-10T18:16:39 EET" finished-at="2020-10-10T18:16:39 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=UserSignUpWithValidData status=SUCCESS method=SignUpTest.UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenshotOnFailure -->
        <test-method status="PASS" signature="UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb]" name="UserSignUpWithValidData" duration-ms="36420" started-at="2020-10-10T18:16:39 EET" data-provider="ExcelData" finished-at="2020-10-10T18:17:15 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[test]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[user]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[3.456798E7]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[test9999@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[password3456]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UserSignUpWithValidData -->
        <test-method status="PASS" signature="screenshotOnFailure(org.testng.ITestResult)[pri:0, instance:testlogic.SignUpTest@5f282abb]" name="screenshotOnFailure" is-config="true" duration-ms="1" started-at="2020-10-10T18:17:15 EET" finished-at="2020-10-10T18:17:15 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=UserSignUpWithValidData status=SUCCESS method=SignUpTest.UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenshotOnFailure -->
        <test-method status="PASS" signature="UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb]" name="UserSignUpWithValidData" duration-ms="37027" started-at="2020-10-10T18:17:15 EET" data-provider="ExcelData" finished-at="2020-10-10T18:17:52 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[test1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[user1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[5678000.0]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[test8888@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[ps7890]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UserSignUpWithValidData -->
        <test-method status="PASS" signature="screenshotOnFailure(org.testng.ITestResult)[pri:0, instance:testlogic.SignUpTest@5f282abb]" name="screenshotOnFailure" is-config="true" duration-ms="1" started-at="2020-10-10T18:17:52 EET" finished-at="2020-10-10T18:17:52 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=UserSignUpWithValidData status=SUCCESS method=SignUpTest.UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenshotOnFailure -->
        <test-method status="FAIL" signature="UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb]" name="UserSignUpWithValidData" duration-ms="7583" started-at="2020-10-10T18:17:52 EET" data-provider="ExcelData" finished-at="2020-10-10T18:18:00 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[test2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[user2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[2.3456923E7]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[wert@dfghj]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[qwer543]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.phptravels.net/account/] but found [https://www.phptravels.net/register]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.phptravels.net/account/] but found [https://www.phptravels.net/register]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testlogic.SignUpTest.UserSignUpWithValidData(SignUpTest.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UserSignUpWithValidData -->
        <test-method status="PASS" signature="screenshotOnFailure(org.testng.ITestResult)[pri:0, instance:testlogic.SignUpTest@5f282abb]" name="screenshotOnFailure" is-config="true" duration-ms="408" started-at="2020-10-10T18:18:00 EET" finished-at="2020-10-10T18:18:00 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=UserSignUpWithValidData status=FAILURE method=SignUpTest.UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenshotOnFailure -->
        <test-method status="FAIL" signature="UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb]" name="UserSignUpWithValidData" duration-ms="7797" started-at="2020-10-10T18:18:00 EET" data-provider="ExcelData" finished-at="2020-10-10T18:18:08 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[test3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[user3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[2.3456712E7]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[test777@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[r2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.phptravels.net/account/] but found [https://www.phptravels.net/register]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.phptravels.net/account/] but found [https://www.phptravels.net/register]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testlogic.SignUpTest.UserSignUpWithValidData(SignUpTest.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UserSignUpWithValidData -->
        <test-method status="PASS" signature="screenshotOnFailure(org.testng.ITestResult)[pri:0, instance:testlogic.SignUpTest@5f282abb]" name="screenshotOnFailure" is-config="true" duration-ms="406" started-at="2020-10-10T18:18:08 EET" finished-at="2020-10-10T18:18:08 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=UserSignUpWithValidData status=FAILURE method=SignUpTest.UserSignUpWithValidData(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:testlogic.SignUpTest@5f282abb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenshotOnFailure -->
        <test-method status="PASS" signature="CloseDriver()[pri:0, instance:testlogic.SignUpTest@5f282abb]" name="CloseDriver" is-config="true" duration-ms="1471" started-at="2020-10-10T18:18:08 EET" finished-at="2020-10-10T18:18:10 EET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseDriver -->
      </class> <!-- testlogic.SignUpTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
